<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Content</class>
 <widget class="QWidget" name="Content">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Template</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#NameInputFrame &gt; * {
	background: transparent;
	border: none;
	font: 10pt &quot;Fira Code&quot;;
	color: #ccc;
}
#ExtLabel {
	color: #aaa;
}
QLabel {
	color: #bbb;
	font: 8pt &quot;Inter&quot;;
}
#CreateBtn {
	padding: 10px;
	min-width: 100px;
	font: 15pt &quot;Inter&quot;;
	background: #575757;
	border-radius: 10px;
	color: white;
}
#CreateBtn:hover {
	background: #666;
}
#CreateBtn:pressed {
	background: #777;
}
#NameInputFrame {
    background: #141414;
    max-height: 40px;
    min-height: 40px;
    border-radius: 10px;
    border: 1px solid #555;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="NameInputLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>File name</string>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <property name="indent">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="NameInputFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="NameInput"/>
         </item>
         <item>
          <widget class="QLabel" name="ExtLabel">
           <property name="text">
            <string>.json</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="CreateBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Create Template</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
